homework 2/homework2/src/homework2/EditOrderStatusServlet.java

```diff
@@ -26,16 +30,59 @@ public EditOrderStatusServlet() {
 	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse response)
 	 */
 	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-		// TODO Auto-generated method stub
-		response.getWriter().append("Served at: ").append(request.getContextPath());
+		int id = Integer.parseInt(request.getParameter("id"));
+		List<Order> orders = (List<Order>) getServletContext().getAttribute("orders");
+		Order leEntry = null;
+		for (Order entry: orders) {
+			if (entry.getId() == id) {
+				leEntry = entry;
+			}
+		}
+		response.setContentType("text/html");
+		PrintWriter out = response.getWriter();
+		out.println("<h1>Editing Order Statuses</h1>");
+		out.println("<form method=\"post\">");
+		out.println("Created: " + leEntry.getCreated() + "'/></br>");
+		out.println("Item:" + leEntry.getItems() + "</br>");
+		out.println("Customer: " + leEntry.getCustname() + "'/></br>");
+		out.println("Status: <input name='status' type='option'" + leEntry.getcompletedstatus() + "'/></br>");
```

Comment:
Plus, you will need to use form to pass information to doPost; otherwise, you will not be able to edit items.